packages/docs/package.json

```diff
@@ -38,8 +38,12 @@
     "@docusaurus/preset-classic": "2.0.0-alpha.43",
     "@docusaurus/theme-live-codeblock": "2.0.0-alpha.43",
     "@docusaurus/utils": "2.0.0-alpha.43",
+    "@requestnetwork/types": "0.12.0",
```

Comment:
@rittme should I include the dependencies I need in the doc here, when using aurelius?

Reply:
for now we're not running them, so it will work without. But, yes, long term this is the right approach.